"-ldl" link problem in Linux

以下のエラーが出てリンクに失敗します
```
jubatus/server/fv_converter/libjubaserv_fv_converter.so: error: undefined reference to 'dlclose'
jubatus/server/fv_converter/libjubaserv_fv_converter.so: error: undefined reference to 'dlerror'
jubatus/server/fv_converter/libjubaserv_fv_converter.so: error: undefined reference to 'dlsym'
jubatus/server/fv_converter/libjubaserv_fv_converter.so: error: undefined reference to 'dlopen'
```

以下のプルリクの変更により，wafのDLが設定されなくなりました．
https://github.com/jubatus/jubatus/pull/1099/files#diff-dc6134e3d3c869a052520d86f46371dcL83
その結果，以下のwscriptのuseにてDLを参照しているにも関わらず，
DLが設定されていないので，リンクの設定が正しくなされなくなったものと思われます．
https://github.com/jubatus/jubatus/blob/master/jubatus/server/fv_converter/wscript#L32

0.9.0のバイナリパッケージには libdl へのリンクが含まれているのですが，
Gentoo上でのビルド時に上記問題が発生しました．
もしかしたらディストリ依存かもしれませんが，上記の原因を見る限りだと
`conf.check_cxx(lib = 'dl')` は必要だと思います．

--- a/wscript
+++ b/wscript
@@ -61,9 +61,6 @@ def options(opt):
 def configure(conf):
   conf.env.CXXFLAGS += ['-O2', '-Wall', '-g', '-pipe', '-pthread'];
   conf.env.LINKFLAGS += ['-pthread']
-#  if sys.platform.startswith("freebsd"):
-#    conf.env.CXXFLAGS += ['-I/usr/local/include']
-#    conf.env.LINKFLAGS += ['-L/usr/local/lib']
 
   conf.load('compiler_cxx')
   conf.load('unittest_gtest')
@@ -84,6 +81,7 @@ def configure(conf):
   conf.check_cxx(lib = 'msgpack')
   conf.check_cxx(lib = 'jubatus_mpio')
   conf.check_cxx(lib = 'jubatus_msgpack-rpc')
+  conf.check_cxx(lib = 'dl', mandatory = False)
 
   # pkg-config tests
   conf.find_program('pkg-config') # make sure that pkg-config command exists

